apps/condo/domains/resident/constants.js

```diff
@@ -144,6 +144,11 @@ const PAYMENT_CATEGORIES_META = [
     },
 ]
 
+const REDIS_LAST_SEND_BILLING_RECEIPT_NOTIFICATION_CREATED_AT_NAME = 'LAST_SEND_BILLING_RECEIPT_NOTIFICATION_CREATED_AT'
```

Comment:
It's some tecnical staff which is useless for anyone. Please, remove it from the file. There is no chances that will reuse it.

Reply:
removed